### DISCUSSION TITLE
Change the name of the field for login

### DISCUSSION THREAD

**Discussion by nnnLik** [fastapi-users/fastapi-users#1240] [Q&A]
I configured fastapi_users exactly as in the documentation, but what really bothers me is that in docs for login you have to send username field which is email. Can I change this field?

**Comment by frankie567** [comment]
The problem here is that we use the built-in [OAuth2PasswordRequestForm](https://github.com/tiangolo/fastapi/blob/d6fb9429d21c6492b2541b7210ecc9707b8677e2/fastapi/security/oauth2.py#L13), which defined by FastAPI to generate the correct form in Swagger. If we don't use it, we lose the auth integration in Swagger.
